src/Popover/Popover.js

```diff
@@ -172,26 +172,33 @@ class Popover extends Component {
 
   renderLayer = () => {
     const {
-      animated, // eslint-disable-line no-unused-vars
+      animated,
       animation,
       children,
       style,
       ...other,
     } = this.props;
 
-    let Animation = animation || PopoverAnimationDefault;
     let styleRoot = style;
 
-    if (!Animation) {
-      Animation = Paper;
+    if (!animated) {
       styleRoot = {
         position: 'fixed',
       };
+
       if (!this.state.open) {
         return null;
       }
+
+      return (
+        <Paper style={Object.assign({}, styleRoot, style)}>
```

Comment:
It would be great to spread the other properties so users can add `id`, `className` or whatever:
```jsx
<Paper style={Object.assign({}, styleRoot, style)} {...other}>
```